src/Phinx/Migration/CreationInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Phinx
+ *
+ * (The MIT license)
+ * Copyright (c) 2014 Rob Morgan
```

Comment:
You need to do that in a LOT of places!!! There are 30 "2014 Rob Morgan"s in the source. :-)